src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const input = document.querySelectorAll('input');
```

Comment:
```suggestion
const inputs = document.querySelectorAll('input');
```
Тут же будет много инпутов